Makefile

```
@@ -100,6 +100,7 @@ build-collector: clean-collector      ## Build the collector SQL scripts.
 	@mkdir -p $(BUILD_DIR)/collector/postgres/sql/
 	@cp scripts/collector/postgres/sql/*.sql $(BUILD_DIR)/collector/postgres/sql
 	@cp scripts/collector/postgres/collect-data.sh $(BUILD_DIR)/collector/postgres/
+        @cp scripts/collector/postgres/db-machine-specs.sh $(BUILD_DIR)/collector/postgres/
```

```suggestion
	@cp scripts/collector/postgres/db-machine-specs.sh $(BUILD_DIR)/collector/postgres/
```

This should be a tab instead of spaces. I think.